spec/features/filter_books_by_genre_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.feature "Filter Books by Genre", type: :feature do
+  # TODO what about filter by "No Genre"?
```

Comment:
Annotation keywords like `TODO` should be all upper case, followed by a colon, and a space, then a note describing the problem.